CODE_OF_CONDUCT.md

```diff
@@ -2,6 +2,9 @@
 SPDX-FileCopyrightText: 2023 Siemens AG
 
 SPDX-License-Identifier: MIT
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
```

Comment:
PR contains mix of formatting/header and fixes we should avoid doing this all in one PR makes is really hard to review

Reply:
I know.